CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Change log
 -----------
 
+* Switch beaglebone-black, beaglebone-green and beaglebone-green-wifi to using poky morty [Florin]
```

Comment:
The official documentation, when referring to the version, seems to style these as "Poky" and "Morty". Lowercase is used when referring to the git branch.